app/serializers/post_action_type_serializer.rb

```diff
@@ -46,11 +46,12 @@ def name_key
   end
 
   def enabled
-    !!PostActionType.enabled_flag_types[object.id]
+    # flags added by API are always enabled
+    true
   end
 
   def applies_to
-    Array.wrap(PostActionType.applies_to[object.id])
+    Flag.valid_applies_to_types
```

Comment:
To simplify, flags added by API can be applied to Post, Topics and Chat Messages (if chat enabled)